portality/tasks/ingestarticles.py

```diff
@@ -71,7 +77,7 @@ def ftp_callback(chunk):
             file_upload.failed("The file at the URL was too large")
             job.add_audit_message("too large")
             try:
-                os.remove(path)
+                os.remove(path) # don't keep this file around
             except:
                 pass
```

Comment:
Add a log statement here. (I know it's not what you were modifying in this PR, but it is needed for improving our ability to troubleshoot these jobs.)

Or maybe even record the exception in the job as an audit message? That would need a bit more work to do sensibly, if you still want the job to be "success" on file removal exceptions.